src/plugins/dashboard_embeddable_container/public/embeddable_plugin.ts

```diff
@@ -17,4 +17,4 @@
  * under the License.
  */
 
-export * from '../../../../../embeddable_api/public/np_ready/public';
+export * from '../../../plugins/embeddable/public';
```

Comment:
Why are we doing this internal re-export, rather than import from `plugins/embeddable/public` where needed?